sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/BeamFnDataClient.java

```diff
@@ -46,7 +46,7 @@
       Endpoints.ApiServiceDescriptor apiServiceDescriptor,
```

Comment:
update method comment or considering naming the object FnDataConsumer

Reply:
Updated